resources/lang/es.json

```diff
@@ -0,0 +1,167 @@
+{
+  "\/about": "\/о нас",
```

Comment:
Russian word??

Reply:
Where?

Reply:
I saw it. It is the first word. Did not see it before. I will fix it.